app/helpers/publishers_helper.rb

```diff
@@ -96,6 +96,29 @@ def publisher_converted_overall_balance(publisher)
     result
   end
 
+  def qualifies_for_payout?(publisher)
+    publisher.selected_wallet_provider_type == Publisher::UPHOLD_CONNECTION && has_minimum_usd_for_payout?(publisher)
```

Comment:
The caller already has `publisher.selected_wallet_provider_type == Publisher::UPHOLD_CONNECTION`. I think you should completely remove this block and move `has_minimum_usd_for_payout?(publisher)` to the caller's block

Reply:
Done.